modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/select_structure_field.jsp

```diff
@@ -92,10 +92,6 @@ portletURL.setParameter("eventName", eventName);
 					ddmField.setDefaultLocale(themeDisplay.getLocale());
 					ddmField.setDDMStructureId(ddmStructureId);
 					ddmField.setName(name);
-
-					if (name.equals(ddmStructureFieldName)) {
-						ddmField.setValue(themeDisplay.getLocale(), ddmStructureFieldValue);
-					}
```

Comment:
If these statements are not related to the current bugs please leave them unchanged.